### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gsklee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ jsonFile(filename, function (err, file) {
 			baz: true
 		}
 	});
+	
+	file.remove('description'); // remove a specific key-value pair
```
Comment (ljharb) [inline README.md]: perhaps this should show both the callback and promise version, like `get` above it?

Comment (gsklee) [inline README.md]: Yep, sure.

**Comment by ljharb** [comment]
If you could rebase this down to a single commit, it's good to go!

**Comment by gsklee** [comment]
I've sent a modified PR in #18 (I'm too lazy to pull the repo down and rebase :joy_cat: )

**Comment by ljharb** [comment]
:-( thanks, in the future please do pull it down rather than making a duplicate PR.

**Pull request closed without merging**